### PR TITLE
New Term: NumPy `.sum()`

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

New keyword entry

### Type of Change

- New Entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/sum/sum.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+`.sum()` provides the following arguments:
+
+- `a`: the array of elements to sum.
+- `axis` : An int or tuple of ints specifying the axis/axes along which to sum.
+- `dtype` : The type of the returned array and the accumulator used to sum elements. Defaults to the dtype of `array`.
+- `out` : An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as expected output.
+- `keepdims` : A boolean if true will keep reduced axes in the result as dimensions with size one.
+- `inital` : Starting value for sum.
+- `where` : A boolean array that maps to `array` and selects elements to include into the sum.
```
Comment (Dusch4593) [inline content/numpy/concepts/built-in-functions/terms/sum/sum.md:35]:
```suggestion
- `axis`: An int or tuple of ints specifying the axis/axes along which to sum.
- `dtype`: The type of the returned array and the accumulator used to sum elements. Defaults to the dtype of `array`.
- `out`: An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as expected output.
- `keepdims`: A boolean if true will keep reduced axes in the result as dimensions with size one.
- `inital`: The starting value for sum.
- `where`: A boolean array that maps to `array` and selects elements to include into the sum.
```

File: content/numpy/concepts/built-in-functions/terms/sum/sum.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+This produces the following output:
+
+```shell
+21
+
+[5 7 9]
+
+[ 6 15]
```
Comment (Dusch4593) [inline content/numpy/concepts/built-in-functions/terms/sum/sum.md:58]:
```suggestion
21
[5 7 9]
[ 6 15]
```

File: content/numpy/concepts/built-in-functions/terms/sum/sum.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+The `.sum()` function sums the elements of an array over a given axis.
+
+## Syntax
+
+```pseudo
+numpy.sum(a, axis, dtype, out, keepdims, inital, where)
+```
+
+All arguments except `a`, the array of elements to sum, are optional. Using `a` by itself will result in a scalar that sums all the elements of the array.
```
Comment (Dusch4593) [inline content/numpy/concepts/built-in-functions/terms/sum/sum.md:25]:
```suggestion
The `a` argument is required and represents the array of elements to sum. All other arguments are optional. Used by itself, `a` will result in a scalar that sums all the elements of the array.
```

File: content/numpy/concepts/built-in-functions/terms/sum/sum.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+`.sum()` provides the following arguments:
+
+- `a`: the array of elements to sum.
```
Comment (Dusch4593) [inline content/numpy/concepts/built-in-functions/terms/sum/sum.md:29]:
```suggestion
- `a`: The array of elements to sum.
```

File: content/numpy/concepts/built-in-functions/terms/sum/sum.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+- `axis` : An int or tuple of ints specifying the axis/axes along which to sum.
+- `dtype` : The type of the returned array and the accumulator used to sum elements. Defaults to the dtype of `array`.
+- `out` : An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as expected output.
+- `keepdims` : A boolean if true will keep reduced axes in the result as dimensions with size one.
+- `inital` : Starting value for sum.
+- `where` : A boolean array that maps to `array` and selects elements to include into the sum.
+
+## Example
+
+The following example creates an array then uses some `.sum` operations to sum the elements.
```
Comment (Dusch4593) [inline content/numpy/concepts/built-in-functions/terms/sum/sum.md:39]:
```suggestion
The following example creates an array then uses a few `.sum()` operations to sum the elements.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review and left some comments/suggestions.

**Comment by SSwiniarski** [comment]
@Dusch4593, I applied your changes.

File: content/numpy/concepts/built-in-functions/terms/sum/sum.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- `out`: An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as expected output.
+- `keepdims`: A boolean if true will keep reduced axes in the result as dimensions with size one.
+- `inital`: The starting value for sum.
+- `where`: A boolean array that maps to `array` and selects elements to include into the sum.
+
+## Example
+
+The following example creates an array then uses a few `.sum()` operations to sum the elements.
+
+```python
```
Comment (Dusch4593) [inline content/numpy/concepts/built-in-functions/terms/sum/sum.md:41]:
```suggestion
```py
```

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski Just one tiny suggestion but LGTM otherwise 👍🏻

**Comment by SSwiniarski** [comment]
@Dusch4593, done!

**Review by KTom101 — APPROVED** [review]
@SSwiniarski No changes from me.

cc @Dusch4593

**Pull request merged**